### ISSUE TITLE
Can't install v0.8.0 - Cannot find module '../global/certbot-dns-plugins'

### ISSUE THREAD

**Issue by tiberiushunter** [hassio-addons/addon-nginx-proxy-manager#117]
# Problem/Motivation

Can't start NPM after upgrading to the new v0.8.0 (to use custom SSL certificates), I've tried uninstalling and reinstalling the addon a couple of times too in case it was my setup but I still get hit by an error

## Expected behavior

Start up NPM

## Actual behavior

Get an error relating to not finding the `certbot-dns-plugins` module

```[20:30:00] INFO: Starting the Manager...
[10/24/2020] [9:30:03 PM] [Global   ] › ✖  error     Cannot find module '../global/certbot-dns-plugins'
Require stack:
- /opt/nginx-proxy-manager/internal/certificate.js
- /opt/nginx-proxy-manager/internal/proxy-host.js
- /opt/nginx-proxy-manager/internal/report.js
- /opt/nginx-proxy-manager/routes/api/reports.js
- /opt/nginx-proxy-manager/routes/api/main.js
- /opt/nginx-proxy-manager/app.js
- /opt/nginx-proxy-manager/index.js Error: Cannot find module '../global/certbot-dns-plugins'
Require stack:
- /opt/nginx-proxy-manager/internal/certificate.js
- /opt/nginx-proxy-manager/internal/proxy-host.js
- /opt/nginx-proxy-manager/internal/report.js
- /opt/nginx-proxy-manager/routes/api/reports.js
- /opt/nginx-proxy-manager/routes/api/main.js
- /opt/nginx-proxy-manager/app.js
- /opt/nginx-proxy-manager/index.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:965:15)
    at Function.Module._load (internal/modules/cjs/loader.js:841:27)
    at Module.require (internal/modules/cjs/loader.js:1025:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (/opt/nginx-proxy-manager/internal/certificate.js:16:26)
    at Module._compile (internal/modules/cjs/loader.js:1137:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1157:10)
    at Module.load (internal/modules/cjs/loader.js:985:32)
    at Function.Module._load (internal/modules/cjs/loader.js:878:14)
    at Module.require (internal/modules/cjs/loader.js:1025:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/opt/nginx-proxy-manager/internal/certificate.js',
    '/opt/nginx-proxy-manager/internal/proxy-host.js',
    '/opt/nginx-proxy-manager/internal/report.js',
    '/opt/nginx-proxy-manager/routes/api/reports.js',
    '/opt/nginx-proxy-manager/routes/api/main.js',
    '/opt/nginx-proxy-manager/app.js',
    '/opt/nginx-proxy-manager/index.js'
  ]
}
```

## Steps to reproduce

Install/Upgrade to v0.8.0

## Proposed changes

I've had a quick look in the repo and I've seen [this PR](https://github.com/jc21/nginx-proxy-manager/pull/664) which is merged in to their develop branch ~6 days ago but not yet released, this looks like it runs the check to see if it exists, if not then runs the installer for it - might be wrong my guess is we just need to wait for that to be released to stop it from erroring without trying to install the certbot-dns-plugins manually? 

edit: linting, typos

**Comment by tiberiushunter** [comment]
Comparing differences between [2.5.0 and 2.6.1](https://github.com/jc21/nginx-proxy-manager/compare/v2.5.0...v2.6.1) I can see this was added as an import: 

![image](https://user-images.githubusercontent.com/10655940/97093776-cc893d00-1646-11eb-9037-8dc9a7ac743c.png)

**Comment by frenck** [comment]
Fixed in ⬆️ NGINX Proxy Manager v0.8.1

https://github.com/hassio-addons/addon-nginx-proxy-manager/releases/tag/v0.8.1
